### DISCUSSION TITLE
Updating Alf.io

### DISCUSSION THREAD

**Discussion by tobsowo** [alfio-event/alf.io#1260] [Deployment]
Please guide on updating Alf.io.

Thank you.

**Comment by tobsowo** [comment]
I'll appreciate some help @cbellone

**Comment by cbellone** [comment]
I need more information about your setup. The update procedure might be different.
How have you installed alf.io?

**Reply by tobsowo** [reply]
I followed this guide: https://gist.github.com/aadhilpm/132b272fcdd4a57b375d80178cce9a98

**Reply by cbellone** [reply]
from what I understand from the tutorial, you should have everything under `/home/alfio`:

- download the [latest version](https://github.com/alfio-event/alf.io/releases/download/2.0-M4-2304/alfio-2.0-M4-2304-boot.war) from our [Releases](https://github.com/alfio-event/alf.io/releases) page
- copy the war file within the `/home/alfio` folder
- update the `start.sh` reflecting the new file name
- `sudo systemctl restart alfio`

**Reply by tobsowo** [reply]
Thank you

**Reply by tobsowo** [reply]
The steps worked as listed.
Thank you.
